src/main/java/net/greatstart/model/GenericResponse.java

```diff
@@ -0,0 +1,18 @@
+package net.greatstart.model;
+
+public class GenericResponse {
+
+    private String message;
+    private String error;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "error" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)